HaxeManual/10-std.tex

```diff
@@ -336,7 +336,7 @@ \subsection{Integer Math}
 	var intDivision = Std.int(6.2/4.7);
 \end{lstlisting}
 
-\todo{I think C++ can use integer operatins, but I don't know about any other targets. Only saw this mentioned in an old discussion thread, still true?}
+\todo{I think C++ can use integer operations, but I don't know about any other targets. Only saw this mentioned in an old discussion thread, still true?}
```

Comment:
Please don't worry about fixing `\todo`, they don't show up on the webpage, and many are rather outdated anyway.